src/test/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdCrosswalkTest.java

```diff
@@ -438,6 +439,206 @@ public void spatialPoint() throws Exception {
     }
 
     @Test
+    public void spatialPolygon() throws Exception {
+        // @formatter:off
+        String ddm = "<?xml version='1.0' encoding='utf-8'?><ddm:DDM" +
+            "  xmlns:ddm='http://easy.dans.knaw.nl/schemas/md/ddm/'" +
+            "  xmlns:gml='http://www.opengis.net/gml'" +
+            "  xmlns:dcx-gml='http://easy.dans.knaw.nl/schemas/dcx/gml/'>" +
+            " <ddm:dcmiMetadata>" +
+            "   <dcx-gml:spatial>" +
+            "     <Polygon xmlns='http://www.opengis.net/gml' srsName='http://www.opengis.net/def/crs/EPSG/0/4326'>" +
```

Comment:
could you add a RD example as well? 
maybe the same triangle, so we see it convert to the same coordinates in datacite?